### PR TITLE
Re-init Jekyll from USWDS-jekyll starter

### PR DESCRIPTION
**Pull request by sknep**
Brian helped me set up this PR that shows the changes much more cleanly. I'll admit to the commit messages being not the best, but Brian said that it makes sense.

### CODE REVIEW

**Comment by sknep** [comment]
Brian and I retcon-ed this so it looks better, per the [dev guide](https://github.com/cityofaustin/developer-guide/blob/master/git/index.md)

**Comment by sknep** [comment]
@briaguya @jcbpl I'd like to go forward with this one. It doesn't require bourbon or neat, but doesn't give us access to the USWDS modules in our own scripts. I can live with that trade for now.

File: _glossary/fence.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (briaguya) [inline _glossary/fence.md]: How is localization being handled with glossary terms?

Comment (sknep) [reply _glossary/fence.md:1]: currently, not at all. Good question.

**Pull request merged**